build-uclibc.sh

```diff
@@ -288,36 +289,45 @@ fi
 # make will fail if there is yet no .config file, but we can ignore this error.
 make distclean >> "${logfile}" 2>&1 || true 
 
-# Patch the temporary install directories used into the uClibc config.
-# uClibc 0.9.34 onwards use defconfig
-if [ ! -f extra/Configs/defconfigs/arc/defconfig ]
+# Copy the defconfig file to a temporary location
+TEMP_DEFCFG=`mktemp --tmpdir=${DEFCFG_DIR} XXXXXXXXXX_defconfig`
```

Comment:
@anthony-kolesov I have a patch that replaces mktemp (both uses) with a custom function that provides the same functionality.  I'm just testing this now, and will raise a pull request shortly.